app/src/main/kotlin/io/homeassistant/companion/android/settings/views/SettingsRow.kt

```diff
@@ -25,13 +25,55 @@ import androidx.compose.ui.unit.dp
 import com.mikepenz.iconics.compose.Image
 import com.mikepenz.iconics.typeface.IIcon
 
+/**
```

Comment:
You could add screenshot tests here too.

Reply:
In this file or ...? I already added screenshot tests for the settings row.